strawberryfields/apps/sample.py

```diff
@@ -86,9 +86,25 @@
 
 The subgraphs sampled from GBS are likely to be dense :cite:`arrazola2018using`, motivating their
 use within heuristics for problems such as maximum clique (see :mod:`~.apps.clique`).
+
+Including node weights
+----------------------
+
+Some graphs are composed of nodes with weights. These node weights can correspond to relevant
+information in an optimization problem and it is desirable to encode them into GBS along with the
+graph's adjacency matrix. One canonical approach to doing this is to use the ``WAW`` encoding
+:cite:`banchi2019molecular`, which rescales the adjacency matrix according to:
+
+.. math::
+
+    A \rightarrow WAW,
+
+with :math:`W` the diagonal matrix formed by the weighted nodes. The rescaled adjacency matrix
+can be passed to :func:`sample`, resulting in a bias toward sampling subgraphs with a large
```

Comment:
I would rephrase this as: 
can be passed to sample, resulting in a distribution that increases the probability of observing a node proportionally to its weight.

Reply:
Thanks have added